app/views/content/steps-to-become-a-teacher/_step_1_check_your_qualifications.html.erb

```diff
@@ -2,6 +2,8 @@
 
 <p>You also need a degree in any subject to teach primary and secondary – if you have one or an equivalent qualification, you can do postgraduate teacher training.</p>
 
-<p>If you do not have a degree, you can do undergraduate teacher training to get a degree alongside qualified teacher status (QTS).</p>
+<p>If you do not have a degree, you can do undergraduate teacher training to get a degree alongside qualified teacher status (QTS). Or you may be eligible to do a <a href=" https://educationhub.blog.gov.uk/2024/02/04/teacher-degree-apprenticeships-how-they-work-and-when-to-apply/"> Teacher Degree Apprenticeship (TDA)</a> and work in a school, earning a salary while getting a batchelor’s degree and QTS.</p>
+
+<p>Applications for the new TDAs are expected to launch in autumn 2024 for both primary and secondary and training is due to start in autumn 2025.</p>
```

Comment:
I'm not sure we should include this line on the Steps page - feels like too much detail and distracting from other info?

Reply:
Sorry for not being clear- I was just questioning the applications open line - feels like a bit too much detail for this page, given that the vast, vast majority of people using the page will be going down the postgrad route

Reply:
and the applications open info is in the linked blog